NETCORE/src/Microsoft.ApplicationInsights.AspNetCore/JavaScriptSnippet.cs

```diff
@@ -55,23 +55,37 @@ public class JavaScriptSnippet : IJavaScriptSnippet
         /// <returns>JavaScript code snippet with instrumentation key or empty if instrumentation key was not set for the application.</returns>
         public string FullScript
         {
-            // TODO: NEED TO SUPPORT CONNECTION STRING. DISCUSS WITH JAVASCRIPT SDK TO CONFIRM COMPATIBILITY.
-
             get
             {
-                if (!this.telemetryConfiguration.DisableTelemetry &&
-                    !string.IsNullOrEmpty(this.telemetryConfiguration.InstrumentationKey))
+                if (!this.telemetryConfiguration.DisableTelemetry)
                 {
+                    string insertConfig;
+                    if (!string.IsNullOrEmpty(this.telemetryConfiguration.ConnectionString))
+                    {
+                        insertConfig =  string.Format(CultureInfo.InvariantCulture, "connectionString: '{0}'", this.telemetryConfiguration.ConnectionString);
+                    }
+                    else if (!string.IsNullOrEmpty(this.telemetryConfiguration.InstrumentationKey))
+                    {
+                        insertConfig = string.Format(CultureInfo.InvariantCulture, "instrumentationKey: '{0}'", this.telemetryConfiguration.InstrumentationKey);
+                    }
+                    else
+                    {
+                        return string.Empty;
```

Comment:
Just curious, what happens when this is reached? Would it be different from the old behavior? Looks like previously passes ikey as empty string and doesn't "crash". Breeze will reject the telemetry, but I think this is okay. If neither is provided, then we "crash".

Reply:
I preserved the same behavior as before: 
- previously, there was the single if statement: If Disabled & ikey empty, returns `string.Empty`
- now with the tested ifs, if Disabled & ikey empty & connString empty, returns `string.Empty'